viYarn/Episode/view/EpisodeViewController.swift

```diff
@@ -8,14 +8,24 @@
 
 import UIKit
 
-class EpisodeTableViewController: UITableViewController {
+class EpisodeViewController: UIViewController, EpisodeLoaderDelagate {
+    
+    lazy var tableView: UITableView = {
```

Comment:
p0: naming: let's call it episodesTableView. Table view doesn't tell anything. What if we had three of those here?